apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -494,3 +496,71 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
         "weight": 1,
         "feedback": {"correct": (score == 1), "missing": {}, "matches": matches},
     }
+
+
+def test(element_html: str, data: pl.ElementTestData) -> None:
+    element = lxml.html.fragment_fromstring(element_html)
+    name = pl.get_string_attrib(
+        element, "answers-name", defaults.element_defaults["answers-name"]
+    )
+    # Get raw correct answer
+    a_tru = data["correct_answers"][name]
```

Comment:
This breaks for elements that don't have answers. The error is reproducible with `demo/drawing/liftingMechanism` from the example course.

Reply:
Not sure how you tested this, as #11867 isn't in yet

Reply:
Manually editing the Python with your fixes from that PR 🙂

Reply:
Oh my 😅

Reply:
I'm assuming you've now tested with this question (with the fixes) and things work?

Reply:
Confirmed.